### PR TITLE
Splash Screen Timing & Decorations

### PR DESCRIPTION
**Pull request by happyrobotstudio**
- Shorten splash timout to 3200ms
- Remove splash window titlebar

### CODE REVIEW

**Comment by ctrlcctrlv** [comment]
I disagree with the changes and recommend this pull request be rejected. If you don't like the splash, no need to shorten it, it can be disabled in the settings. It's always been 7 seconds.

**Comment by skef** [comment]
Code looks fine to me, I don't have strong feelings about the change otherwise.

**Comment by happyrobotstudio** [comment]
Sweeet, I like the borderless and quick(er) approach

Seems to have been some to and fro about this, but yep, hopefully the usage of wam_nodecor is correct

File: fontforgeexe/startui.c [hunk 1]
```diff
@@ -393,12 +393,14 @@ exit( 0 );		/* But the event loop should never return */
...
 static void start_splash_screen(void){
+
     GDrawSetVisible(splashw,true);
+
     GDrawSync(NULL);
     GDrawProcessPendingEvents(NULL);
     GDrawProcessPendingEvents(NULL);
 
-    splasht = GDrawRequestTimer(splashw,7000,1000,NULL);
+    splasht = GDrawRequestTimer(splashw,3200,1000,NULL);
```
Comment (ctrlcctrlv) [inline fontforgeexe/startui.c]: If this were changed back to 7000 I'd have no problems with this, but as it stands I can't accept it.

**Comment by happyrobotstudio** [comment]
> I disagree with the changes and recommend this pull request be rejected. If you don't like the splash, no need to shorten it, it can be disabled in the settings. It's always been 7 seconds.

Ahh I see, it seems strange to me as a user of Photoshop for example, the purp

File: fontforgeexe/startui.c (hunk 1 above)
Comment (happyrobotstudio) [reply fontforgeexe/startui.c]: Have addressed in #3795

**Comment by happyrobotstudio** [comment]
Thanks all for quick review !

Yep 7 seconds is nice for splash, the image is pleasing =)

But yes I would very much enjoy the title bar being absent,

All the best,

- Daniel

**Comment by ctrlcctrlv** [comment]
Please push a commit changing it back; then I'll test it, and if it works well, merge it.

**Comment by happyrobotstudio** [comment]
> Please push a commit changing it back; then I'll test it, and if it works well, merge it.

Ready at #3795

**Pull request closed without merging**